compiler/passes/convert-uast.cpp

```diff
@@ -1763,6 +1773,9 @@ struct Converter {
     bool zippered = node->iterand()->isZip();
     bool isForExpr = node->isExpressionLevel();
 
+    // convert these for now, despite the error, so that symbols are converted.
+    convertWithClause(node->withClause(), node);
```

Comment:
Just wondering what the error is.

Reply:
Currently, we do not allow with-clauses on foreach loops. Without this change to call ``convertWithClause`` the elements of that with-clause would be unconverted and cause an internal error. Doing this was easier than trying to create some kind of "ignore specific unconverted symbols" system.

Reply:
Thanks!